stwcs/wcsutil/headerlet.py

```diff
@@ -1248,7 +1248,7 @@ def attach_headerlet(filename, hdrlet, logging=False, logmode='a'):
 
 @with_logging
 def delete_headerlet(filename, hdrname=None, hdrext=None, distname=None,
-                     logging=False, logmode='w'):
+                     delete_all=False, logging=False, logmode='w'):
```

Comment:
The parameter name of _delete_all_ is consistent precisely because the original docstring described this operation as **deleting all** the headerlet extensions with the same distname.  I am actually inclined to add 'deletes all' to the NOTES for _hdrname_ as well to emphasize the actual default behavior, with this parameter modifying that default behavior.  The original docstring was actually what was misleading, since it implied that only a single extension would be deleted if a _hdrname_ was specified, which in fact was not correct as we found.  That is the reason for this PR.

Reply:
Fixed in 9098e78